File: pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float v;
    do
    {
        v = round(get_float("Value: ")*100);
    }
    while (v <= 0);

    int coins = 0;
    while (true)
    {
        if (v >= 25)
        {
            v -= 25;
            coins++;
        }
        else if (v >= 10)
        {
            v -= 10;
            coins++;
        }
        else if (v >= 5)
        {
            v -= 5;
            coins++;
        }
        else if (v >= 1)
        {
            v -= 1;
            coins++;
        }
        else
        {
            break;
        }
    }
    printf("Coins: %i\n", coins);
}

File: pset6/sentimental-cash/cash.py
from cs50 import get_float

while True:
    v = get_float("Change owed: ") * 100
    if v > 0:
        break

coins = 0
while True:
    if v >= 25:
        v -= 25
        coins += 1
    elif v >= 10:
        v -= 10
        coins += 1
    elif v >= 5:
        v -= 5
        coins += 1
    elif v >= 1:
        v -= 1
        coins += 1
    else:
        break

print(coins)